src/DestroyCommand.php

```diff
@@ -21,9 +21,8 @@ protected function configure()
     /**
      * Execute the command.
      *
-     * @param  \Symfony\Component\Console\Input\InputInterface  $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
-     * @return void
```

Comment:
Not sure, I copied ```.php_cs``` from framework to my homestead folder and
ran ```php-cs-fixer fix src```

On Sun, Jul 5, 2015 at 2:19 PM, Graham Campbell <notifications@github.com>
wrote:

> In src/DestroyCommand.php
> <https://github.com/laravel/homestead/pull/226#discussion_r33897973>:
>
> > @@ -21,9 +21,8 @@ protected function configure()
> >      /**
> >       * Execute the command.
> >       *
> > -     * @param  \Symfony\Component\Console\Input\InputInterface  $input
> > -     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
> > -     * @return void
>
> You probably ran it with the symfony level by mistake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/homestead/pull/226/files#r33897973>.
>



-- 
- Joe Ferguson
MemphisTechnology.org
MemphisPHP.org
HACKMemphis.com